src/main/java/org/spongepowered/common/statistic/SpongeStatisticGroup.java

```diff
@@ -0,0 +1,161 @@
+import org.spongepowered.api.text.format.TextStyles;
+import org.spongepowered.api.text.translation.Translation;
+import org.spongepowered.common.registry.RegistryHelper;
+import org.spongepowered.common.text.translation.SpongeTranslation;
+
+import java.util.Map;
+
+public class SpongeStatisticGroup implements StatisticGroup, TextRepresentable {
+
+    // Happens too early for SpongeRegistry
+    private static boolean initialized = false;
+
+    public static synchronized void init() {
+        if (!initialized) {
+            initialized = true;
+            Map<String, StatisticGroup> groups = Maps.newLinkedHashMap();
+            groups.put("general", of("GENERAL", "sponge:general", "stat.generalButton"));
+            groups.put("hidden", of("HIDDEN", "sponge:hidden", "stats.tooltip.type.achievement"));
+            groups.put("has_killed_entity", of("HAS_KILLED_ENTITY", "sponge:hasKilledEntity", "stat.entityKills"));
+            groups.put("killed_by_entity", of("KILLED_BY_ENTITY", "sponge:killedByEntity", "stat.entityKilledBy"));
+            groups.put("craft_item", of("CRAFT_ITEM", "sponge:craftItem", "stat.crafted"));
+            groups.put("use_item", of("USE_ITEM", "sponge:useItem", "stat.used"));
+            groups.put("break_item", of("BREAK_ITEM", "sponge:breakItem", "stat.depleted"));
+            groups.put("craft_block", of("CRAFT_BLOCK", "sponge:craftBlock", "stat.crafted"));
+            groups.put("use_block", of("USE_BLOCK", "sponge:useBlock", "stat.used"));
+            groups.put("mine_block", of("MINE_BLOCK", "sponge:mineBlock", "stat.mined"));
+            // TODO: Add Translation or remove them entirely
+            groups.put("has_killed_team", of("HAS_KILLED_TEAM", "sponge:hasKilledTeam", ""));
+            groups.put("killed_by_team", of("KILLED_BY_TEAM", "sponge:killedByTeam", ""));
+            RegistryHelper.mapFields(StatisticGroups.class, groups);
+        }
+    }
+
+    private static StatisticGroup of(String id, String name, String translation) {
+        return new SpongeStatisticGroup(checkNotNull(id, "id").toUpperCase(), checkNotNull(name, "name"),
+                new SpongeTranslation(checkNotNull(translation, "translation")));
+    }
+
+    private final String id;
+    private final String name;
+    private final Translation translation;
+
+    public SpongeStatisticGroup(String id, String name, Translation translation) {
+        super();
+        this.id = checkNotNull(id, "id");
+        this.name = checkNotNull(name, "name");
+        this.translation = checkNotNull(translation, "translation");
+    }
+
+    @Override
+    public final String getId() {
+        return this.id;
+    }
+
+    @Override
+    public final String getName() {
+        return this.name;
+    }
+
+    @Override
+    public final Translation getTranslation() {
+        return this.translation;
+    }
+
+    @Override
+    public StatisticFormat getDefaultStatisticFormat() {
+        // TODO: Misconception?
```

Comment:
Which is always the same.... (I just consider it hillarious to check whether the group  has the same format and return absent in that case just to ask the group for that value. Which itself is a const)

````java
stat.getFormat().orElse(stat.getGroup().getDefaultFormat());
// => (besides being horrible to read)
stat.format == stat.getGroup().format ? stat.getGroup().format : stat.format
````
Which is always the same as `stat.format`.
Furthermore there is no benefit in knowing the groups default or at least I can think of any usecase.